WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -330,8 +330,10 @@ extension WordPressOrgXMLRPCApi {
                 return
             }
 
+            var evaluationError: CFError?
+            _ = SecTrustEvaluateWithError(serverTrust, &evaluationError)
             var result = SecTrustResultType.invalid
-            let certificateStatus = SecTrustEvaluate(serverTrust, &result)
+            let certificateStatus = SecTrustGetTrustResult(serverTrust, &result)
```

Comment:
Again, I don't think improving this code is necessarily in scope for this PR, but I noticed two things.

- `evaluationError` is unused. I guess we don't get a warning from the compiler because of the assignment in line 334, but the code itself never reads it. Is there any kind of error handling that we should do here? Maybe previous authors have more insight on the kind of treatment this code deserves? cc @jleandroperez @astralbodies 
- While there is a test for the class (🎉) it might not be enough to cover this scenario. I overrode `result` with `.invalid` after this line and the test still passed.

Reply:
Right, I was also experimenting with it, but decided to keep exact same logic as before. Any additional input on this would be nice!
Removed unused error var in 4669e9f671bede92c2be24cb8c2e9940d83c315e.

Reply:
Hey there folks! I'm afraid I can't recall working on this code, too many years :(

That being said, if there's a Trust evaluation error, I doubt there's much we can do. Fallback to "default handling", and the URL Request should fail IMO